src/app/dataset-view/additional-components/dataset-settings-component/tabs/compacting/dataset-settings-compacting-tab.component.ts

```diff
@@ -0,0 +1,81 @@
+import { NavigationService } from "./../../../../../services/navigation.service";
+import { ChangeDetectionStrategy, Component, Input, OnInit } from "@angular/core";
+import { FormBuilder, Validators } from "@angular/forms";
+import { RxwebValidators } from "@rxweb/reactive-form-validators";
+import { DatasetBasicsFragment, DatasetFlowType } from "src/app/api/kamu.graphql.interface";
+import { promiseWithCatch } from "src/app/common/app.helpers";
+import { CompactingTooltipsTexts } from "src/app/common/tooltips/compacting.text";
+import { ModalService } from "src/app/components/modal/modal.service";
+import { SliceUnit, sliceSizeMapper } from "./dataset-settings-compacting-tab.types";
+import { DatasetCompactingService } from "../../services/dataset-compacting.service";
+import { DatasetViewTypeEnum } from "src/app/dataset-view/dataset-view.interface";
+import AppValues from "src/app/common/app.values";
+
+@Component({
+    selector: "app-dataset-settings-compacting-tab",
+    templateUrl: "./dataset-settings-compacting-tab.component.html",
+    styleUrls: ["./dataset-settings-compacting-tab.component.scss"],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class DatasetSettingsCompactingTabComponent implements OnInit {
+    @Input() public datasetBasics: DatasetBasicsFragment;
+    public hardCompactionForm = this.fb.group({
+        sliceUnit: [SliceUnit.GB, [Validators.required]],
```

Comment:
Let's ask SM about the expected defaults. I think 1GB might be a bit too heavyweight slice for transfer protocol parallelism. I think the optimal value is likely in tens of megabytes.

Reply:
### +